ScriptingExample/build.gradle

```diff
@@ -1,28 +1,28 @@
-// Note: "common.gradle" in the root project contains additional initialization
-//   for this project. This initialization is applied in the "build.gradle"
-//   of the root project.
+plugins {
+	id 'zencode-common'
+}
 
-// NetBeans will automatically add "run" and "debug" tasks relying on the
-// "mainClass" property. You may however define the property prior executing
-// tasks by passing a "-PmainClass=<QUALIFIED_CLASS_NAME>" argument.
-//
-// Note however, that you may define your own "run" and "debug" task if you
-// prefer. In this case NetBeans will not add these tasks but you may rely on
-// your own implementation.
 if (!hasProperty('mainClass')) {
 	ext.mainClass = 'org.openzen.zenscript.scriptingexample.Main'
 }
 
+configurations {
+	register("stdLibs") {
```

Comment:
Why the register call here instead of directly using its name like in other build scripts?
Alternatively, why the opposite in all the other build scripts?